template/page-events.html.jinja

```diff
@@ -499,19 +498,38 @@
     "image": "AMAs.png",
     "end_date": "October 17, 2023"
     },
-]
-%}
-    {# 
-    Needs Links
     { "name": "XRPL Accelerator Demo Day",      
     "description": "Join us for XRPL Accelerator Demo Day in Singapore! Explore pitches from 11 promising startups building on the XRP Ledger, network with founders and investors, and kickstart the Singapore FinTech Festival.",
     "type": "meetup",
-    "link": "",
-    "location": "Hybrid (In-person: Singapore Central Business District. Virtual attendance: Webinar link)",
+    "link": "https://www.eventbrite.co.uk/e/xrpl-demo-day-tickets-740650023157?aff=oddtdtcreator",
+    "location": "Hybrid Singapore/Virtual Webinar",
     "date": "November 14, 2023",
     "image": "SouthKoreaMeetup.png",
     "end_date": "November 14, 2023"
     },
+    { "name": "XRPL Blockhack Hackathon",      
+    "description": "Join us at George Brown College's Waterfront Campus for workshops and talks on promoting growth for blockchain projects and ventures. We are supporting a for the most innovative application built on XRPL.",
+    "type": "hackathon",
+    "link": "https://blockhack-2023.devpost.com/",
+    "location": "George Brown College - Waterfront Campus",
+    "date": "October 20, 2023 - October 22, 2023",
+    "image": "Hackathons.png",
+    "end_date": "October 22, 2023"
```

Comment:
This event has already ended, so should not be adding it.

Reply:
@amarantha-k It will go to the past events section.